devito/operator.py

```diff
@@ -93,10 +100,17 @@ def apply(self, debug=False):
         return tuple([param.data for param in self.output_params])
 
     def apply_python(self):
+        """Uses Python to apply the operator
+
+        :returns: A tuple containing the values of the operator outputs
+        """
         self.run_python()
         return tuple([param.data for param in self.output_params])
 
     def find_free_terms(self, expr):
+        """:param expr: The expression to search
```

Comment:
I know the name says it all, but for consistency a one-liner at the top would be nice.